docker/cloud/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.20-bullseye AS builder
+WORKDIR /app
+COPY . .
+COPY ./docker/cloud/config.toml .
+RUN go build -v -o nitro .
+
+FROM debian:bullseye-slim
+RUN apt-get update
+RUN apt-get install -y ca-certificates
+RUN rm -rf /var/lib/apt/lists/*
+WORKDIR /app/
+COPY --from=builder /app/nitro .
+COPY --from=builder /app/config.toml .
+
+EXPOSE 3005 4005
```

Comment:
@bitwiseguy was it a conscious decision to not expose the gui port 5005?

Reply:
Yes it was a conscious decision, but its one we can certainly change. I was trying to keep the image as small as possible so I excluded all of the react code. The main reason to exclude that code for now is that Digital Ocean free tier allows 1 container registry, 500MB of storage (next tier is $5/month for 5 repos, 5GB)

Reply:
Ok good to know. Do we know how heavy the react part would be? Might be nice to have that available as a quick-and-dirty dashboard for the cloud deployment.

My hunch is that it adds a negligible amount of weight to the image.

Reply:
```
~/s/go-nitro ❯❯❯ go build -o gonitro && go build -o gonitro-with-ui -tags embed_ui &&  ls -l gonitro*                                                                                                              main ✭ ◼
-rwxr-xr-x  1 georgeknee  staff  59115138 14 Aug 17:59 gonitro
-rwxr-xr-x  1 georgeknee  staff  59716818 14 Aug 17:59 gonitro-with-ui
```

Reply:
I created a new issue to make this change. Since this PR has been dangling around for awhile and has grown quite large, I'd like to get it merged and address with a follow-up PR. https://github.com/statechannels/go-nitro/issues/1536